storage/innobase/ibuf/ibuf0ibuf.cc

```diff
@@ -2367,6 +2367,10 @@ static void ibuf_delete_recs(const page_id_t page_id)
 static void ibuf_read_merge_pages(const uint32_t* space_ids,
 				  const uint32_t* page_nos, ulint n_stored)
 {
+	/* Identify slow shutdown cleanup phase to clean up change buffers. */
+	bool slow_shutdown_cleanup = !srv_fast_shutdown
+				     && srv_shutdown_state >= SRV_SHUTDOWN_CLEANUP;
```

Comment:
There only ever was one change buffer. In the archaic formatting style, overflow lines are indented by only 1 extra TAB.

Reply:
I think you prefer "change buffer elements" or "change buffer entries" here. It seems I was not correct assuming it can be intuitively deduced. Either way this lines are now removed and "slow_shutdown_cleanup" is now passed down as function parameter.